src/models/converters/transactions/mod.rs

```diff
@@ -181,6 +200,7 @@ impl MultisigTransaction {
                 .data_decoded
                 .as_ref()
                 .and_then(|it| it.get_action_count()),
+            to_info: info_provider.address_info(&self.to).ok(), //TODO test
```

Comment:
similar here, could we move it right next to the `to` similar to how it is in the `ModuleTransaction`?